lib/sync_checker/formats/edition_base.rb

```diff
@@ -26,6 +26,7 @@ def edition_expected_in_draft
       end
 
       def edition_expected_in_live
+        return nil if latest_edition_is_rejected?(document)
```

Comment:
Could we write it as this?

```ruby
case
when document.published_edition
  document.published_edition
when document.pre_publication_edition && document.pre_publication_edition.rejected?
  nil
when document.pre_publication_edition && document.pre_publication_edition.unpublishing
  document.pre_publication_edition
end
```

That way we don't need to go looking for the method definition to check the behaviour